test/NuGet.Core.Tests/NuGet.Commands.Test/IncludeTypeTests.cs

```diff
@@ -1860,6 +1992,40 @@ private static void CreateXYZ(string repositoryDir, string include, string exclu
             SimpleTestPackageUtility.CreatePackages(packages, repositoryDir);
         }
 
+        private static void CreateXyzIndividually(string repositoryDir, string include, string exclude)
+        {
+            var z = new SimpleTestPackageContext()
+            {
+                Id = "packageZ",
+                Version = "1.0.0",
+                Include = include,
+                Exclude = exclude
+            };
+
+            var y = new SimpleTestPackageContext()
+            {
+                Id = "packageY",
+                Version = "1.0.0",
+                Include = include,
+                Exclude = exclude
+            };
+
+            var x = new SimpleTestPackageContext()
+            {
+                Id = "packageX",
+                Version = "1.0.0",
+                Include = include,
+                Exclude = exclude
+            };
+
+            var packages = new List<SimpleTestPackageContext>()
+            {
+                x, y, z
+            };
+
+            SimpleTestPackageUtility.CreatePackages(packages, repositoryDir);
```

Comment:
note: you could do SimpleTestPackageUtility.CreatePackages(repositoryDir, x, y, z); 
no need to change it, but there are helpers for this common scenario